### ISSUE TITLE
error while compile

### ISSUE THREAD

**Issue by sniperxnl** [YannikW/IR-Laptimer#2]
Arduino: 1.6.6 (Windows 8.1), Board:"NodeMCU 1.0 (ESP-12E Module), 160 MHz, Serial, 115200, 4M (3M SPIFFS)"

Build-opties gewijzigd, alles wordt opnieuw gebuild
C:\Users\Bas\Documents\Open-Lap-master (1)\Open-Lap-master\Receiver\Software\RX_ESP_V06\RX_ESP_V06.ino: In function 'void setup()':

RX_ESP_V06:166: error: 'initiateCommands' was not declared in this scope

   initiateCommands();

```
                ^
```

RX_ESP_V06:169: error: 'connectWifi' was not declared in this scope

   wifiConnected = connectWifi();  //Connect to WiFi if possible and save which WiFi is used 

```
                           ^
```

C:\Users\Bas\Documents\Open-Lap-master (1)\Open-Lap-master\Receiver\Software\RX_ESP_V06\RX_ESP_V06.ino: In function 'void loop()':

RX_ESP_V06:240: error: 'resetValue' was not declared in this scope

```
           resetValue(sensor);

                            ^
```

RX_ESP_V06:251: error: 'resetValue' was not declared in this scope

```
       resetValue(sensor);

                        ^
```

RX_ESP_V06:267: error: 'getValue' was not declared in this scope

```
     message[sensor][a] = getValue(buf[sensor][a]);

                                                 ^
```

RX_ESP_V06:284: error: 'twoOfFour' was not declared in this scope

```
       if(twoOfFour(sensor, data[sensor]) && (actTime >= (lastPilotTime[data[sensor]] + MINLAPTIME)))  //New lap

                                        ^
```

RX_ESP_V06:286: error: 'newLap' was not declared in this scope

```
         newLap(data[sensor]);

                            ^
```

RX_ESP_V06:303: error: 'saveValue' was not declared in this scope

```
       saveValue(sensor, data[sensor]);

                                     ^
```

RX_ESP_V06:331: error: 'resetValue' was not declared in this scope

```
   resetValue(sensor);

                    ^
```

C:\Users\Bas\Documents\Open-Lap-master (1)\Open-Lap-master\Receiver\Software\RX_ESP_V06\RX_ESP_V06.ino: In function 'void newLap(uint8_t)':

RX_ESP_V06:424: error: 'beep' was not declared in this scope

```
 beep(BEEPLENGTH);

                ^
```

C:\Users\Bas\Documents\Open-Lap-master (1)\Open-Lap-master\Receiver\Software\RX_ESP_V06\Commands.ino: In function 'void initiateCommands()':

Commands:18: error: 'sayVersion' was not declared in this scope

   SCmd.addCommand("VERSION", sayVersion);       //Print Software Version

```
                          ^
```

Commands:19: error: 'setDebugmode' was not declared in this scope

   SCmd.addCommand("DEBUGMODE", setDebugmode);   //Set no debugging, simple debugging, or full debugging

```
                            ^
```

Commands:22: error: 'fpvRaceTracker' was not declared in this scope

   SCmd.addCommand(rtm, fpvRaceTracker);         //Start FPV Race Tracker operation mode

```
                    ^
```

Commands:23: error: 'resetRace' was not declared in this scope

   SCmd.addCommand("NEW_OPEN_LAP", resetRace);   //Reset all laps, times, ... 

```
                               ^
```

Commands:25: error: 'help' was not declared in this scope

   SCmd.addCommand("HELP", help);                //Show possible commands

```
                       ^
```

Commands:26: error: 'unrecognized' was not declared in this scope

   SCmd.addDefaultHandler(unrecognized);         //Unknown Command input

```
                      ^
```

Commands:29: error: 'easyRace' was not declared in this scope

   TCPCmd.addCommand("RESET#", easyRace);          //Start EasyRaceLapTimer operation mode  

```
                           ^
```

Commands:30: error: 'fpvRaceTrackerTCP' was not declared in this scope

   TCPCmd.addCommand(rtm, fpvRaceTrackerTCP);      //Start FPV Race Tracker TCP operation mode

```
                      ^
```

exit status 1
'initiateCommands' was not declared in this scope

  Dit rapport zou meer informatie hebben met
  "Tijdens de compilatie uitgebreide uitvoer weergeven"
  ingeschakeld in Bestand > Voorkeuren.

**Comment by YannikW** [comment]
Thats weired, if I compile it it works without problems. Do you have installed the EPS8266 library? And when you open the `RX_ESP_V06.ino`, are there four tabs opened in Arduino? They should be `RX_ESP_V06.ino`, `Commands.ino`, `SerialCommand.cpp` and `SerialCommand.h`.

Your error says all the functions and some variables are not declared, but they are :D 
If all above is ok, you should try to update your Arduino IDE to the newest version.

**Comment by sniperxnl** [comment]
When i try the Version 5 from you then it works great
![2](https://cloud.githubusercontent.com/assets/16321561/11876327/6e7728f4-a4e9-11e5-8bfc-d7fa06ca2c28.PNG)

**Comment by YannikW** [comment]
That's very strange.
You told me you use Arduino V1.0.6.5 - newest Version is 1.6.6 ( https://www.arduino.cc/en/Main/Software ) Please try this version (That's the one I use)

**Comment by sniperxnl** [comment]
![2](https://cloud.githubusercontent.com/assets/16321561/11876506/64040e90-a4ea-11e5-8872-c8d30e09beac.PNG)

**Comment by YannikW** [comment]
Ok, I've seen I use 1.6.5. I updated to 1.6.6 (and 1.6.7 - it was released just an hour ago) and with this versions I get the same error. 

I will try to fix the files for use them with 1.6.7. If you want to use them this moment you have to downgrade to 1.6.5 ;)

**Comment by YannikW** [comment]
Code is fixed. V0.6.1 works now with Arduino 1.6.7 (and should work with 1.6.6, too)
